cfme/tests/cloud/test_quota.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from riggerlib import recursive_update
+
+from widgetastic.utils import partial_match
+
+from cfme import test_requirements
+from cfme.base.credential import Credential
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.utils.generators import random_vm_name
+
+
+pytestmark = [
+    test_requirements.quota,
+    pytest.mark.long_running,
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([AzureProvider], scope='function',
+                         required_fields=[["provisioning", "image"]])]
+
+
+@pytest.fixture
+def prov_data(appliance, provisioning):
+    instance_type = "d2s_v3" if appliance.version < "5.10" else "D2s_v3"
+    return {
+        "catalog": {"vm_name": random_vm_name(context="quota")},
```

Comment:
This prov_data isn't actually doing any setup, consider just using a non-fixture function here, or embedding this in the test directly.

Reply:
@ganeshhubale thanks for explanations, sounds good!

Please add a docblock to the fixture explaining that second bullet point - that you're in a fixture here because you're relying on provisioning, and just returning a dictionary.